src/core/data.tf

```diff
@@ -169,6 +169,11 @@ resource "azurerm_monitor_metric_alert" "cosmos_api_throttling_alert" {
     webhook_properties = {}
   }
 
+  action {
+    action_group_id    = data.azurerm_monitor_action_group.io_com_action_group.id
+    webhook_properties = {}
+  }
```

Comment:
I would remove this `action`. 

IMHO we should define our own `azurerm_monitor_metric_alert` in https://github.com/pagopa/io-messages with the query configured to look only for our collections (`messages`, `messages-view`, `messages-status` ecc.)

Reply:
https://github.com/pagopa/io-infra/pull/1164/commits/ef797ff60c9c4ced53a1be97d219e7f5261a4363